### ISSUE TITLE
QList Side Icons squashed with fontawesome fa-folder

### ISSUE THREAD

**Issue by mstaack** [quasarframework/quasar#1709]
On latest quasar version on chrome,
if enabling fontawesome icons, the `fa-file` icon gets squashed.

(Interestingly this does not happen with`fa-folder`)

This code:
```
        <q-list highlight inset-separator>
            <q-list-header inset>Files</q-list-header>
            <q-item>
                <q-item-side icon="fa-folder" inverted color="grey-6"/>
                <q-item-main>
                    <q-item-tile label>Photos</q-item-tile>
                    <q-item-tile sublabel>February 22, 2016</q-item-tile>
                </q-item-main>
                <q-item-side right icon="fa-info-circle"/>
            </q-item>
        </q-list>
        <q-list>
            <q-list-header inset>Folders</q-list-header>
            <q-item>
                <q-item-side icon="folder" inverted color="primary"/>
                <q-item-main>
                    <q-item-tile label>Photos</q-item-tile>
                    <q-item-tile sublabel>February 22, 2016</q-item-tile>
                </q-item-main>
                <q-item-side right icon="info" color="green"/>
            </q-item>
            <q-item>
                <q-item-side icon="fa-file" inverted color="primary"/>
                <q-item-main>
                    <q-item-tile label>Photos</q-item-tile>
                    <q-item-tile sublabel>February 22, 2016</q-item-tile>
                </q-item-main>
                <q-item-side right icon="info" color="green"/>
            </q-item>
        </q-list>
```

Results into this:
![image](https://user-images.githubusercontent.com/10169509/37048432-2f6ac630-216e-11e8-9492-09ccbdbf1bb6.png)

**Comment by pdanpdan** [comment]
Please include a forked JsFiddle/Codepen from one of these:
- https://jsfiddle.net/rstoenescu/waugrryy/ (jsFiddle - Material theme) or
- https://jsfiddle.net/rstoenescu/7gu065yg/ (jsFiddle - iOS theme) or
- https://codepen.io/rstoenescu/pen/KQRZJg (Codepen - Material theme) or
- https://codepen.io/rstoenescu/pen/paVpBN (Codepen - iOS theme)

**Comment by pdanpdan** [comment]
It looks like it's a problem from FA css definition.

**Comment by rstoenescu** [comment]
Available in 0.15.7
